### PR TITLE
[WJ-854] Replace FTML base CSS with a different system in ftml-components

### PR DESCRIPTION
**Pull request by Monkatraz**
This PR replaces the base CSS found in `ftml/misc/ftml-base.css` with SCSS found in `ftml-components`. This is so that CSS can be organized more cleanly, and also use some of the infrastructure we have around SCSS, such as Stylelint. In the process of doing this, I've removed FTML's (the crate) handling of the base CSS file. You're going to need to import `ftml-components` to use FTML anyways, so I think this is fine.

### CODE REVIEW

**Comment by Monkatraz** [comment]
I just realized I saved the `README.MD` with Prettier on, which I doubt hurts anything, but that's why there is formatting changes.

File: ftml/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 ### Foundation Text Markup Language
 
 A Rust library to parse Wikidot text ("Wikitext") into an abstract syntax tree (AST).
-This aims to be a replacement for the aging [Text\_Wiki](https://github.com/gabrys/wikidot/tree/master/lib/Text_Wiki/Text) from Wikidot.
+This aims to be a replacement for the aging [Text_Wiki](https://github.com/gabrys/wikidot/tree/master/lib/Text_Wiki/Text) from Wikidot.
```
Comment (emmiegit) [inline ftml/README.md:18]: Really? This parses as a warning on my editor, which is why I added the escape. I didn't realize Prettier cared about this.

**Review by emmiegit — APPROVED** [review]

File: ftml/README.md (hunk 1 above)
Comment (Monkatraz) [reply ftml/README.md:18]: I think Prettier tries to get rid of any escapes that aren't strictly needed? This was weird to me as well, but I do know some details about the CommonMark spec and I can't see any reason why that `_` would actually need to be escaped.

**Comment by Monkatraz** [comment]
thanks @ammongit 

I'm probably doing footnotes next, because that'll help fill out some infrastructure needed for lightweight tooltips (Tippy.js is a fairly large package, so I might make my own).

**Pull request merged**